### PR TITLE
Replace aarch64 with arm64 in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/25060

### CODE REVIEW

**Review by berenss — APPROVED** [review]

**Review by berenss — CHANGES_REQUESTED** [review]
Do not make changes to the release_images - they need to stay as aarch64

**Comment by berenss** [comment]
@gurnben @dtthuynh @jaywcarman please keep me honest here.

1. when we're talking about install-config.yaml options, in the controlPlane.architecture field it needs to be arm64
2. when we're talking about release images and the clusterImageSets we need to reference aarch64

**Comment by oafischer** [comment]
Got it @berenss, thanks for pointing that out. I reverted all the changes in release_images.

File: clusters/create_vm.adoc [hunk 1]
```diff
@@ -50,7 +50,9 @@ The release image identifies the version of the {ocp-short} image that is used t
 
 *Note:* Only release images for {ocp-short} versions 4.5.x and higher are supported.
 
-The node pools include the control plane pool and the worker pools. The control plane nodes share the management of the cluster activity. The information includes the _Architecture_ field. If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+The node pools include the control plane pool and the worker pools. The control plane nodes share the management of the cluster activity. The information includes the _Architecture_ field:
```
Comment (dockerymick) [inline clusters/create_vm.adoc]: It seems like there should be a period at the end of this instead of a colon

File: clusters/create_vm.adoc [hunk 2]
```diff
@@ -50,7 +50,9 @@ The release image identifies the version of the {ocp-short} image that is used t
...
+
+* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (dockerymick) [inline clusters/create_vm.adoc:58]: I recommend prompting the user for the definition of the _Architecture_ field. Add something along the lines of "View the following field description:"

 I took a look at the doc and i recommend we add some subsections. For example, after the first Tip, I would name add a subsection titled, **Adding your cluster to an existing cluster set**. Idk it just seems daunting to read and the information is cluttered in my opinion.

File: clusters/create_virtualization.adoc [hunk 3]
```diff
@@ -50,9 +50,9 @@ If there is already a base DNS domain that is associated with the selected crede
...
 
-The information for your node pools includes the number of Cores, Sockets, Memory, and Disk size for the the control plane pool. The three control plane nodes share the management of the cluster activity.
+The information for your node pools includes the number of Cores, Sockets, Memory, and Disk size for the the control plane pool. The three control plane nodes share the management of the cluster activity. The information includes the _Architecture_ field:
 
-If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (dockerymick) [inline clusters/create_virtualization.adoc:59]: I recommend the same suggestion for each file. My suggestion is most likely a different effort and this PR is specifically created to update the value.

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: clusters/create_virtualization.adoc (hunk 3 above)
Comment (oafischer) [reply clusters/create_virtualization.adoc:59]: Is there going to be a conflict? This PR is just addressing this issue: https://github.com/stolostron/backlog/issues/25060

File: clusters/create_azure.adoc [hunk 4]
```diff
@@ -52,7 +52,7 @@ The Node pools include the control plane pool and the worker pools. The control
 
  * *Optional* Region: Specify a region where you want to run your node pools. You can select multiple zones within the region for a more distributed group of control plane nodes. A closer zone might provide faster performance, but a more distant zone might be more distributed.
 
- * *Optional* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+ * *Optional* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (berenss) [inline clusters/create_azure.adoc]: this looks good to me.

File: clusters/create_google.adoc [hunk 5]
```diff
@@ -51,7 +51,7 @@ The Node pools include the control plane pool and the worker pools. The control
 
 * Region: Specify a region where you want to run your control plane pools. A closer region might provide faster performance, but a more distant region might be more distributed.
 
-* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (berenss) [inline clusters/create_google.adoc:59]: looks good to me.

File: clusters/create_ocp_aws.adoc [hunk 6]
```diff
@@ -47,7 +47,7 @@ The release image identifies the version of the {ocp-short} image that is used t
 
 The node pools include the control plane pool and the worker pools. The control plane nodes share the management of the cluster activity. The information includes the following fields:
 
-* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (berenss) [inline clusters/create_ocp_aws.adoc:54]: looks good

File: clusters/create_openstack.adoc [hunk 7]
```diff
@@ -50,7 +50,7 @@ The release image identifies the version of the {ocp-short} image that is used t
 
 The node pools include the control plane pool and the worker pools. The control plane nodes share the management of the cluster activity. The information includes the following fields:
 
-* *Optional* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _aarch64_.
+* *Optional* Architecture: If the architecture type of the managed cluster is not the same as the architecture of your hub cluster, enter a value for the instruction set architecture of the machines in the pool. Valid values are _amd64_, _ppc64le_, _s390x_, and _arm64_.
```
Comment (berenss) [inline clusters/create_openstack.adoc:57]: yes looks good

File: clusters/create_virtualization.adoc (hunk 3 above)
Comment (berenss) [reply clusters/create_virtualization.adoc:59]: looks good to me

File: clusters/create_vm.adoc (hunk 2 above)
Comment (berenss) [reply clusters/create_vm.adoc:58]: looks good to me.

**Review by berenss — APPROVED** [review]
lgtm

File: clusters/create_vm.adoc (hunk 2 above)
Comment (oafischer) [reply clusters/create_vm.adoc:58]: @dockerymick I agree, went ahead and added that subsection. Also added your other suggestion.

File: clusters/create_google.adoc [hunk 8]
```diff
@@ -39,6 +39,9 @@ The name of your cluster is used in the hostname of the cluster. There are some
 
 *Tip:* Select *YAML: On* to view content updates as you enter the information in the console.
 
+[#google_adding-your-cluster-to-existing-cluster-set]
+== Adding your cluster to an existing cluster set
```
Comment (dockerymick) [inline clusters/create_google.adoc:45]: Thanks for adding this! Add the childlink for this section in the TOC at the beginning of the file., where you list the sections within the file

File: clusters/create_virtualization.adoc [hunk 9]
```diff
@@ -50,9 +50,9 @@ If there is already a base DNS domain that is associated with the selected crede
 
 The release image identifies the version of the {ocp-short} image that is used to create the cluster. If the version that you want to use is available, you can select the image from the list of images. If the image that you want to use is not a standard image, you can enter the URL to the image that you want to use. See xref:../clusters/release_images.adoc#release-images[Release images] for more information about release images. 
 
-The information for your node pools includes the number of Cores, Sockets, Memory, and Disk size for the the control plane pool. The three control plane nodes share the management of the cluster activity.
+The information for your node pools includes the number of Cores, Sockets, Memory, and Disk size for the the control plane pool. The three control plane nodes share the management of the cluster activity. The information includes the _Architecture_ field:
```
Comment (dockerymick) [inline clusters/create_virtualization.adoc]: add "View the following field description:" after "The information includes the _Architecture_ field."

File: clusters/create_vm.adoc [hunk 10]
```diff
@@ -40,6 +40,9 @@ The name of your cluster is used in the hostname of the cluster.
 
 *Tip:* Select *YAML: On* to view content updates as you enter the information in the console.
 
+[#vsphere_adding-your-cluster-to-existing-cluster-set]
+== Adding your cluster to an existing cluster set
+
```
Comment (dockerymick) [inline clusters/create_vm.adoc:45]: Update the TOC inn the file by adding the childlink (<<section-anchor,Title of section>>) at the beginning of the file

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made a couple of suggestions

File: clusters/create_openstack.adoc [hunk 11]
```diff
@@ -40,6 +40,9 @@ The name of the cluster is used in the hostname of the cluster. The name must co
 
 *Tip:* Select *YAML: On* to view content updates as you enter the information in the console.
 
+[#openstack_adding-your-cluster-to-existing-cluster-set]
+== Adding your cluster to an existing cluster set
```
Comment (dockerymick) [inline clusters/create_openstack.adoc:45]: same change for this file. Whenever you add a new section in a file, remember to add the childlink in the TOC within the file

File: clusters/create_virtualization.adoc [hunk 12]
```diff
@@ -42,6 +42,9 @@ The name of your cluster is used in the hostname of the cluster.
 
 *Tip:* Select *YAML: On* to view content updates as you enter the information in the console.
 
+[#virtualization-adding-your-cluster-to-existing-cluster-set]
+== Adding your cluster to an existing cluster set
```
Comment (dockerymick) [inline clusters/create_virtualization.adoc:47]: Same

**Comment by oafischer** [comment]
I think I got everything @dockerymick if you wanna give it one final check.

**Comment by oafischer** [comment]
Thanks for catching that one @dockerymick, I did miss that one. Are we good to merge now?

**Review by dockerymick — APPROVED** [review]
All good! Thanks for your patience! Approving

**Pull request merged**